src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/model/or/GroupORModel.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.externalusersapi.model.or
```

Comment:
Need to check whether this model is required

Reply:
NO ACTION - leave as is